test/core/security/credentials_test.cc

```diff
@@ -3598,6 +3598,55 @@ TEST(CredentialsTest, TestFakeCallCredentialsCompareFailure) {
   grpc_call_credentials_release(md_creds);
 }
 
+TEST(CredentialsTest, TestCompositeChannelCredsCompareSuccess) {
```

Comment:
Can we add another test case that tests composite call creds? Just to have a more complete test coverage?

Reply:
Ah got it.